content/docs/user-guide/dvcignore.md

```diff
@@ -28,87 +29,95 @@ Ignored files will not be saved in <abbr>cache</abbr>, they will be non-existent
 for DVC. It's worth to remember that, especially when ignoring files inside
 DVC-handled directories.
 
-**It is crucial to understand, that DVC might remove ignored files upon
-`dvc run` or `dvc repro`. If they are not produced by a
-[pipeline](/doc/command-reference/dag) [stage](/doc/command-reference/run), they
-can be deleted permanently.**
+⚠️ Important! Note that `dvc run` and `dvc repro` might remove ignored files. If
+they are not produced by a pipeline [stage](/doc/command-reference/run), they
+can be lost permanently.
+
+Keep in mind, that when you add `.dvcignore` patterns that affect an existing
+<abbr>output</abbr>, its status will change and DVC will behave as if that
+affected files were deleted.
 
-Keep in mind, that when you add to `.dvcignore` entries that affect one of the
-existing <abbr>outputs</abbr>, its status will change and DVC will behave as if
-that affected files were deleted.
+💡 Note that you can use the `dvc check-ignore` command to check whether given
+files or directories are ignored by the patterns in a `.dvcignore` file.
 
 If DVC finds a `.dvcignore` file inside a dependency or output directory, it
 raises an error. Ignoring files inside such directories should be handled from a
 `.dvcignore` in higher levels of the project tree.
 
-## Syntax
-
-The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
```

Comment:
I think that was useful. Also, we do quite a bit of changes not related directly to the command, @jorgeorpinel :)

Reply:
Yes, sorry. But had to read the whole thing to find the best places to link to the command and fixed copy edits along the way. This section was repetitive since the format (and templates) is already linked to from the bullets above. It was also an entire H2 section for 1-line — looked funny once rendered.

Should I restore it and move all the syntax/format and templates info here? (remove it from the bullet under **How it works**)